ui/desktop/app/controllers/scopes/scope/projects/settings/index.js

```diff
@@ -3,6 +3,68 @@
  * SPDX-License-Identifier: BUSL-1.1
  */
 
-import Controller from '@ember/controller';
+import Controller, { inject as controller } from '@ember/controller';
+import { inject as service } from '@ember/service';
+import { getOwner } from '@ember/application';
+import { action } from '@ember/object';
 
-export default class ScopesScopeProjectsSettingsIndexController extends Controller {}
+const THEMES = [
+  {
+    label: 'system',
+    value: 'system-default-theme',
+  },
+  {
+    label: 'light',
+    value: 'light',
+  },
+  {
+    label: 'dark',
+    value: 'dark',
+  },
+];
+
+export default class ScopesScopeProjectsSettingsIndexController extends Controller {
+  @controller('scopes/scope/projects/settings/index') settings;
```

Comment:
What was this for? Doesn't this refer to the same controller?

Reply:
yup this wasn't needed, I forgot to remove this